src/helper/useFormHandled.jsx

```diff
@@ -3,19 +3,19 @@ import { BasicValidate } from "./personalValidation";
 import _ from "lodash";
 /**
  * Custom hook with logic for form
- * its includes validations, controller values, errorMessage and requireds  
+ * its includes validations, controller values, errorMessage and requireds
  */
 export const useFormHandled = () => {
   // Object with all details on each field
   const [inputs, setInputs] = useState({});
   // Indicate if all form values is valid
   const [isValidForm, setIsValidForm] = useState(true);
   // Each time the inputs changes, checks if the form is valid
-  useMemo(()=>{
+  useMemo(() => {
     for (const field of Object.values(inputs)) {
-        if(field.isValid !== undefined && !field.isValid){
-          return setIsValidForm(false);
-        }
+      if (field.isValid !== undefined && !field.isValid) {
+        return setIsValidForm(false);
+      }
     }
```

Comment:
I still don't know why `useMemo()` - you are not using the returned value from the "create" function

Reply:
also, this file's suffix should be ".js"

Reply:
> I still don't know why `useMemo()` - you are not using the returned value from the "create" function

bump